homeassistant/package_constraints.txt

```diff
@@ -50,8 +50,8 @@ httplib2>=0.19.0
 # gRPC is an implicit dependency that we want to make explicit so we manage
 # upgrades intentionally. It is a large package to build from source and we
 # want to ensure we have wheels built.
```

Comment:
Do we need to do anything to make the wheels build or should just merging this PR be fine?

Reply:
Nothing to do, just merging is fine.

The wheel builder input has been updated to explicitly include grpc explicitly, thenuses the package constraints to decide which version of the wheel to build (https://github.com/home-assistant/core/blob/4e9bc9eaffd464f192d187a01771a86699b2f932/.github/workflows/wheels.yml#L163)